ui/src/main/java/edu/wpi/grip/ui/util/StringInMemoryFile.java

```diff
@@ -22,6 +22,7 @@
    * @param contents    The contents of the file.
    * @param permissions The permissions to use for the file.
    */
+  @SuppressWarnings("PMD.AvoidUsingOctalValues")
```

Comment:
Isn't this where the octal value is?
https://github.com/WPIRoboticsProjects/GRIP/pull/620/files#diff-1ae559d890a9aee6f5aa62c0de38e999R34
Why is this annotation on the other constructor?